### ISSUE TITLE
Shrink map loss stucked

### ISSUE THREAD

**Issue by light42** [MhLiao/DB#350]
I've tried to train resnet50_db++ by paddleocr.
The binary loss(~0.1) and threshold loss(~0.3) decreased significantly, yet only shrink map loss(~0.7) which were stuck at high loss compared to others.

I've tried adjusting learning_rate, weight_decay, unclip_ratio, thresh, box_tresh, shrink_ratio, shrink_thresholds, without any noticeable improvements. In some cases it just make the losses bigger.

Half of my training datasets have width or height that went between 2000-4000 pixels, while the length of bbox could reached 100 pixels.

**Comment by trugoj** [comment]
Hi there -- any progress on your issue? Did you solve it in the end?

**Comment by light42** [comment]
> Hi there -- any progress on your issue? Did you solve it in the end?

Sorry, I almost forgot. Its beyond my ability to debug the model so I stopped working on it. My conclusion is both models and my datasets are not adequate enough.

Closing it since it's not my concern anymore.
